### PR TITLE
Return players if no param

### PR DESCRIPTION
**Pull request by Sarahtonein**
@Balugrizzly  can you please review this and https://github.com/BloomGameStudio/PlayerService/pull/63

### CODE REVIEW

File: controllers/rest/player/GetPlayer.go [hunk 1]
```diff
@@ -16,7 +16,7 @@ func GetPlayer(c echo.Context) error {
 	db := database.GetDB()
 
 	// Read the "active" query parameter from the URL
-	activeParam := c.QueryParam("active")
+	activeParam := c.QueryParam("include_active")
```
Comment (Balugrizzly) [inline controllers/rest/player/GetPlayer.go:19]: Why did you override my change?
can you revert that?

File: controllers/rest/player/GetPlayer.go [hunk 2]
```diff
@@ -38,7 +38,7 @@ func GetPlayer(c echo.Context) error {
 	queryPlayer := &models.Player{}
 	queryPlayer.Active = active
 	players := &[]models.Player{}
-	if err := db.Preload(clause.Associations).Where(queryPlayer.Active).Find(players).Error; err != nil {
+	if err := db.Preload(clause.Associations).Where("active = ?", queryPlayer.Active).Find(players).Error; err != nil {
```
Comment (Balugrizzly) [inline controllers/rest/player/GetPlayer.go:41]: Why is a prepared string statement needed here?

can you use typed where clause

cant you filter by queryPlayer like i did initially?

**Review by Balugrizzly — CHANGES_REQUESTED** [review]

File: controllers/rest/player/GetPlayer.go (hunk 1 above)
Comment (Sarahtonein) [reply controllers/rest/player/GetPlayer.go:19]: Yes will revert, changed it as I thought we were going to use include_active.

File: controllers/rest/player/GetPlayer.go (hunk 2 above)
Comment (Sarahtonein) [reply controllers/rest/player/GetPlayer.go:41]: I tried to filter using this line and was receiving an empty slice:

```if err := db.Preload(clause.Associations).Where(queryPlayer.Active).Find(players).Error; err != nil {
``` 

Does it work on your end?

Comment (Balugrizzly) [reply controllers/rest/player/GetPlayer.go:41]: Mate this line is empty for me 

My line that was there initially worked for me yes 
You added the `.Active` having that alone i can see as not working

Comment (Sarahtonein) [reply controllers/rest/player/GetPlayer.go]: > Mate this line is empty for me
> 
> My line that was there initially worked for me yes You added the `.Active` having that alone i can see as not working

Edited my response, the code I put in my prior comment was on the same line as the backticks :( 

I have updated per your response :)

Comment (Balugrizzly) [reply controllers/rest/player/GetPlayer.go]: Yea was never saying this was not working.
You sent me a blank line and asked if it was working i understood it that it does not work on your end.
I in response to that said i could see the possibility that the .Active is the cause of it. And told you that i know that my line that was there initially is known to work.
 
The issue was the non typed string query.

but kk all good :) good work

**Pull request merged**